controllers/mutation_reconcile_looper.go

```diff
@@ -399,8 +399,9 @@ func (m *MutationReconcileLooper) createSubstitutionRulesForLocalization(ctx con
 			case *localblob.AccessSpec:
 				if x.GlobalAccess == nil {
 					refErr = errors.New("cannot determine image digest")
+				} else {
```

Comment:
The above error was overwritten by the next line without the else, so was an ineffectual assignment.